client/app/admin-dashboard/page.jsx

```diff
@@ -22,7 +22,7 @@ export default async function AdminDashboardView() {
   return (
     <div>
       <Typography variant="h1">Reservations:</Typography>
-      <Suspense fallback={<CircularProgress />}>
+      <Suspense fallback={<Spinner />}>
```

Comment:
Why is using Spinner better than using Circular Progress?
Does it just look better?

Reply:
The Spinner component uses the CircularProgress component within it, it just puts into a Box  for better formatting on the page.